code/HyperbolicModels/app.config

```diff
@@ -1,3 +0,0 @@
-<?xml version="1.0" encoding="utf-8"?>
-<configuration>
-<startup><supportedRuntime version="v4.0" sku=".NETFramework,Version=v4.5"/></startup></configuration>
```

Comment:
What was the reason for the change to app.config?   (maybe not targeting a specific .NET framework?)